clang/cursor.go

```diff
@@ -45,20 +46,54 @@ func (c Cursor) PlatformAvailability(availabilitySize int) (always_deprecated bo
  */
 type CursorVisitor func(cursor, parent Cursor) (status ChildVisitResult)
 
+type functionsMapping struct {
```

Comment:
"funcsMap" is OK. I thought that we might need this more than once. Maybe I should put it in its own file?